src/main/java/net/mcreator/ui/modgui/ModElementGUI.java

```diff
@@ -395,6 +407,8 @@ protected final void finalizeGUI(boolean wrapInScrollpane) {
 			openInEditingMode(generatableElement);
 		}
 
+		SwingUtilities.invokeLater(() -> elementUpdateListener.registerUI(centerComponent, false));
```

Comment:
Keyboard events can also make mod element changes, so not sure why this flag is set to false here

Reply:
This parameter does not affect text fields

Reply:
Then documentation does not explain what it does